SprawdzanieNIPu/ConsoleApplication1/NIP.cs

```diff
@@ -1,72 +1,111 @@
-﻿using System;
+﻿//-----------------------------------------------------------------------
+// <copyright file="NIP.cs" company="CompanyName">
+// Company copyright tag.
+// </copyright>
+//----------------------------------------------------------------------- 
 
 namespace ConsoleApplication1
-{
-    class NIP
-    {
-        public string numer;
+ {
+    using System;
 
-        public NIP(string nr)
-        {
-            this.numer = nr.Trim();
-        }
+    /// <summary>
+    /// Klasa służy do sprawdzenia, czy wprowadzony ciąg znaków jest poprawnym numerem NIP.
+    /// </summary>
+ internal class NIP
```

Comment:
Na oko obiekt NIP nie powinien być możliwy do stworzenia w stanie niepoprawnym. Acz to oczywiście zależy od tego, co ma robić program.